src/VisualStudio/CSharp/Test/ProjectSystemShim/LegacyProject/CSharpCompilerOptionsTests.cs

```diff
@@ -116,20 +119,75 @@ public void ProjectOutputPathAndOutputExeNameChange()
                 project.SetOutputFileName(initialPath);
                 Assert.Equal(initialPath, project.GetOutputFileName());
 
+                var outputs = (CompilationOutputFiles)environment.Workspace.GetCompilationOutputs(project.VisualStudioProject.Id);
+                Assert.Equal(initialPath, outputs.AssemblyFilePath);
+                Assert.Equal(null, outputs.PdbFilePath);
+
                 // Change output folder from command line arguments - verify that objOutputPath changes.
                 var newPath = @"C:\NewFolder\test.dll";
                 project.SetOutputFileName(newPath);
                 Assert.Equal(newPath, project.GetOutputFileName());
 
+                outputs = (CompilationOutputFiles)environment.Workspace.GetCompilationOutputs(project.VisualStudioProject.Id);
+                Assert.Equal(newPath, outputs.AssemblyFilePath);
```

Comment:
That didn't change from the call above? The comment "verify that objOutputPath changes" implies it should have (or the comment is wrong)

Reply:
(the comment might need an update and that's it...)

Reply:
It did change. Used to be `initialPath`, now it's `newPath`

Reply:
Alright, going to the eye doctor.